virt/vm_templates/virt-automatic-bootsource-updates.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="virt-automatic-bootsource-updates"]
+= Managing automatic boot source updates
+include::_attributes/common-attributes.adoc[]
+:context: virt-automatic-bootsource-updates
+
+toc::[]
+
+By default, {VirtProductName} automatically updates system-defined boot sources. You can opt out of automatic updates for all system boot sources by disabling the `enableCommonBootImageImport` feature gate.
+
+Custom boot sources are not controlled by the feature gate. You must manage them individually by editing the `HyperConverged` custom resource (CR). You can also use this method to manage individual system-defined boot sources.
+
```

Comment:
Thanks, @apinnick - this makes sense. I decided to create a concept module about automatic boot source updates because the assembly text was getting lengthy. I tried to incorporate your suggestion, though I didn't want to list the boot sources that RH provides in case it changes (also, I think in OKD the RHEL attribute might render as Fedora).

Reply:
In theory, the auto-updated boot sources should not appear in OKD docs because we do not provide this service upstream. We should probably check at some point to see exactly what is being published in OKD Virt.